app/static/pages/volunteer/Volunteer.js

```diff
@@ -56,17 +66,22 @@ oppiaFoundationWebsite.controller('VolunteerPage', [
         scrollToTopButton.removeClass('scrolling');
       }
     });
+    $scope.buttonDisabled = false;
```

Comment:
What's this for -- is it used? Also, from the name, not sure which button it refers to.